packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -202,15 +202,24 @@ portfolio.addProduct(product);
 
 TagOptions allow administrators to easily manage tags on provisioned products by creating a selection of tags for end users to choose from.
 For example, an end user can choose an `ec2` for the instance type size.
-TagOptions are created by specifying a key with a selection of values.
+TagOptions are created by specifying a key with a selection of values, 
+and can be associated with both portfolios and products.  When launching a product,
+both the TagOptions associated with the product and the containing portfolio are made available.
+
 At the moment, TagOptions can only be disabled in the console.
 
-```ts fixture=basic-portfolio
-const tagOptions = new servicecatalog.TagOptions({
+```ts fixture=portfolio-product
+const tagOptionsForPortfolio = new servicecatalog.TagOptions({
+  costCenter: ['A', 'B'],
+});
+
+const tagOptionsForProduct = new servicecatalog.TagOptions({
   ec2InstanceType: ['A1', 'M4'],
   ec2InstanceSize: ['medium', 'large'],
 });
-portfolio.associateTagOptions(tagOptions);
+
+portfolio.associateTagOptions(tagOptionsForPortfolio);
```

Comment:
can move these to be grouped as tag option - tag option association.

Reply:
Yes please - the current grouping is pretty confusing.